joint_deconvolution/joint_deconvolution_and_feather_auto.py

```diff
@@ -0,0 +1,99 @@
+import json, os, sys, glob
+
+region      = sys.argv[1]
+pth         = os.path.abspath(os.getcwd())
+
+with open(f"./default_tclean_commands.json", "r") as fh:
```

Comment:
if we don't have `default_tclean_commands.json` in the same directory, as I'm suggesting above, we should do something like what we do [elsewhere](https://github.com/ACES-CMZ/reduction_ACES/blob/158e54b4b2f9ca0900832e62755c4981cf977668/retrieval_scripts/run_pipeline.py#L45), i.e.:
```python
rootdir = os.getenv['ACES_ROOTDIR']
with open(f"{rootdir}/pipeline_scripts/default_tclean_commands.json", "r") as fh:
```